index.src.html

```diff
@@ -670,72 +670,75 @@ <h3 id="settings-object">
     Is |settings object| a secure context?
   </h3>
 
-  Given a <a>settings object</a> (|settings object|), this algorithm returns
+  Given a <a>settings object</a>, |settings object|, this algorithm returns
```

Comment:
Nit: I actually prefer keeping the `()`. This is probably just personal style, but I find it significantly easier to distinguish the variable name from the type name if it's segregated via something more clear than ",".